.eslintrc.js

```diff
@@ -184,5 +184,23 @@ module.exports = {
         '@redwoodjs/process-env-computed': 'off',
       },
     },
+    {
+      files: ['**/src/**'],
+      excludedFiles: [
+        '**/__tests__/**',
+        '**/*.test.ts?(x)',
```

Comment:
this copy 🍝  brought to light the fact that `create-redwood-app` includes `*.test.*` files in the npm artifact - which may or may not be what we want (seems as though we do for `api/src/directives/requireAuth/requireAuth.test.{j,t}s` and `api/src/directives/skipAuth/skipAuth.test.{j,t}s` ).

this then begs the questions:

- do we want to exclude `create-redwood-app`?
- are there other test files we maybe want to include with this linter rule?

Reply:
@jtoar see above

Reply:
cc @jtoar  ( fwiw, just about the test files )

Reply:
Excluding test files may make sense for rolling this out easier yeah